### ISSUE TITLE
2FA - Is it possible to implement it?

### ISSUE THREAD

**Issue by marcosrgarcia** [b5prolh/MetamonPlayer#2]
Is it possible to implement a version that is compatible with the 2FA (2 factor autentication) that was released a few weeks ago?

**Thank you for your help!** Ill keep sending eggs and $RACA in your way whenever possible.

**Comment by b5prolh** [comment]
> Is it possible to implement a version that is compatible with the 2FA (2 factor autentication) that was released a few weeks ago?
> 
> **Thank you for your help!** Ill keep sending eggs and $RACA in your way whenever possible.

Yes, it is possible to implement to pass 2FA . But it is not safe for your account because you need setup to read your email to get code. It's not security so I don't do that. Pls turn off 2FA before using tool and open it again after tool complete. It is inconvenient but safe. **Thanks for your donate.**
